clustermesh-apiserver/clustermesh/k8s/resources.go

```diff
@@ -24,8 +27,10 @@ var (
 		"Clustermesh-apiserver Kubernetes resources",
 
 		cell.Config(k8s.DefaultConfig),
+		cell.Config(operator.MCSAPIConfig{}),
 		cell.Provide(
 			k8s.ServiceResource,
+			mcsapi.ServiceExportResource,
```

Comment:
Would it be appropriate to have a warning in case the MCS-API support is enabled, but the ServiceExports CRD is not present?